docs/hardware.md

```diff
@@ -53,19 +56,24 @@ There are some frames that are provided that will provide some base:
 - [Momentum Drones DEV-7](https://momentumdrones.com/products/dev7-frame-kit)
 
 ### Flight controllers
-
-- [Pixhawk 4](https://docs.px4.io/main/en/flight_controller/pixhawk4.html) discontinued, go look at:
-   -  [Holybro's Pixhawk 6C](https://holybro.com/collections/autopilot-flight-controllers/products/pixhawk-6c)
-   -  [CUAV's Pixhawk V6x](https://doc.cuav.net/flight-controller/pixhawk-v6x/en/#building-firmware)
-- [mRo PixRacer R15](https://store.mrobotics.io/product-p/auav-pxrcr-r15-mr.htm) discontinued, go look at:
+-Holybro
```

Comment:
```suggestion
- Holybro
```
There needs to be a space between the hyphen and the word  else it won't render as a list. You can use `mkdocs serve` to check this locally